docs/tutorials/constraints.rst

```diff
@@ -0,0 +1,255 @@
+:orphan:
```

Comment:
This is unnecessary - it was needed in the past when there was no index pointing to the tutorials, but now there is, so you should *not* use ``:orphan:`` (it indicates that the document in question isn't linked to from anywhere else).